test/extended/util/client.go

```diff
@@ -8,6 +8,7 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"k8s.io/client-go/util/retry"
```

Comment:
order!

Reply:
let's see how CI goes, i don't want to risk another e2e failure rush. i'll address in a followup